tests/test_attenuator.py

```diff
@@ -160,3 +160,32 @@ def test_attenuator():
 @pytest.mark.timeout(5)
 def test_attenuator_disconnected():
     AttBase('TST:ATT', name='test_att')
+
+
+@pytest.fixture(
+    params=['at2l0', 'at1k4']
+)
+def fake_new_attenuator(request):
+    """AT2L0, AT1K4 - attenuators new to LCLS-II."""
+    if request.param == 'at2l0':
+        FakeAT2L0 = make_fake_device(AT2L0)
+        return FakeAT2L0('AT2L0:', name='fake_at2l0')
+    FakeAT1K4 = make_fake_device(AT1K4)
+    return FakeAT1K4('AT1K4:', calculator_prefix='AT1K4:CALC',
+                     name='fake_at1k4')
+
+
+def test_new_attenuator_smoke(fake_new_attenuator):
+    fake_new_attenuator.setpoint
+    fake_new_attenuator.readback
+    fake_new_attenuator.actuate
+    fake_new_attenuator(0.0)
+    fake_new_attenuator(1.0)
+    fake_new_attenuator.wm()
+    with pytest.raises(ValueError):
+        fake_new_attenuator(1.1)
+    print(
+        fake_new_attenuator.format_status_info(
```

Comment:
(Doing this is what led me to work on https://github.com/pcdshub/pcdsdevices/pull/740)